pages.ko/common/curl.md

```diff
@@ -0,0 +1,36 @@
+# curl
+
+> 데이터를 서버에서 혹은 서버로 전송. HTTP,FTP 및 POP3를 포함한 대부분의 프로토콜 지원. 
+> 더 많은 정보: [https://curl.haxx.se](<https://curl.haxx.se/>).
```

Comment:
> 데이터를 서버에서 혹은 서버로 전송. 
> HTTP,FTP 및 POP3를 포함한 대부분의 프로토콜 지원. 
> 더 많은 정보: <https://curl.haxx.se/>.

I think this is more better.

Reply:
That's right, this respects the original format! Good catch :slightly_smiling_face: